src/krati_Shukla/Assignment30/MaxSecond.java

```diff
@@ -0,0 +1,25 @@
+package krati_Shukla.Assignment30;
+
+public class MaxSecond {
+
+	void Max(int[] input) {
+
+		for (int index = 0; index < input.length; index++) {
```

Comment:
Take looks of below code
Shashank - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1560/files